versions/src/main/java/cc/polyfrost/oneconfig/utils/commands/arguments/OldPlayerArgumentParser.java

```diff
@@ -0,0 +1,44 @@
+//#if MC<=11202
+package cc.polyfrost.oneconfig.utils.commands.arguments;
+
+import cc.polyfrost.oneconfig.internal.utils.Deprecator;
+import com.google.common.collect.Lists;
+import net.minecraft.client.Minecraft;
+import net.minecraft.entity.player.EntityPlayer;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+
+import java.lang.reflect.Parameter;
+import java.util.List;
+import java.util.stream.Collectors;
+
+/**
+ * The old player argument parser. Returns an {@link EntityPlayer}.
+ * @deprecated Use {@link PlayerArgumentParser} instead.
+ */
+@Deprecated
+public class OldPlayerArgumentParser extends ArgumentParser<EntityPlayer> {
```

Comment:
call this entity player argument parser please

Reply:
Fine